tests/pages/services/ServiceTable-cy.js

```diff
@@ -234,9 +234,9 @@ describe("Service Table", function() {
         .should("to.have.length", 1);
 
       cy
-        .get(".modal pre")
+        .get(".modal-body")
```

Comment:
Actually, I'm not going to remove this because there are others like destroy, scale, that are in the exact same situation with duplicated tests. I don't want to change this one and then have inconsistency with the other modal states.